app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,11 @@
+class DownvotesController < ApplicationController
+  def create
+    if current_user
+      @review = Review.find(params[:review_id])
+      @vote = @review.votes.find_or_create_by(user: current_user)
```

Comment:
this appropriately finds or creates a vote, but how is the value of the vote set?